libtransmission/tr-udp.cc

```diff
@@ -10,6 +10,10 @@
 
 #ifdef _WIN32
 #include <io.h> /* dup2() */
+#ifdef _MSC_VER
+#include <BaseTsd.h>
+typedef SSIZE_T ssize_t; // VS quirk for POSIX ssize_t
+#endif // _MSC_VER
```

Comment:
`::sendto()` is the only place where we're using `ssize_t`. Not a PR blocker, but consider refactoring `tr_session::tr_udp_core::sendto()` to use `auto ret = ::sendto()` to avoid this quirk

Reply:
OK